boards/st/stm32u083c_dk/stm32u083c_dk.dts

```diff
@@ -134,6 +134,12 @@
 	};
 };
 
+stm32_lp_tick_source: &lptim2 {
+	clocks = <&rcc STM32_CLOCK(APB1, 30)>,
```

Comment:
the stm32u0.dtsi has 
  clocks = <&rcc STM32_CLOCK(APB1, 30**U**)>;  
same here ?